extensions/todoist/src/create-task.tsx

```diff
@@ -29,6 +29,12 @@ import { isTodoistInstalled } from "./hooks/useIsTodoistInstalled";
 import { useNLPParser } from "./hooks/useNLPParser";
 import useSyncData from "./hooks/useSyncData";
 
+// Local interface to work around TypeScript language server not recognizing
```

Comment:
It should work, did you `npm run dev` and then restart the TS server in VSCode?

<img width="1362" height="200" alt="CleanShot 2025-08-26 at 09 27 19@2x" src="https://github.com/user-attachments/assets/b142d8d1-df61-434c-b3cb-1602e6e689f9" />